### ISSUE TITLE
Table Insert with from wrong table in supabase do not produce error

### ISSUE THREAD

**Issue by Ejayz** [supabase/supabase-js#1269]
# Bug report

- [ /] I confirm this is a bug with Supabase, not with my own application. 
- [/ ] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

 ```
 const { data, error } = await supabase.from("tbl_transaction").insert({
  }).select();
```
apparantly if your table on from() method is incorrect when inserting data wont return any error . is this by design or bug ? thank you .

lets say instead of **tbl_transaction** you put **transaction_table** then it will just show **error: {}** as return for error and **null** for data .

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Try creating a record with supabase insert with wrong table name.

## Expected behavior

It should return an error for wrong table name inserted .

## System information

- OS:Windows 10
- Browser (if applies) Chrome
- Version of supabase-js: 2.45.2
- Version of Node.js:v20.17.0

**Comment by apferrarone** [comment]
Yes I have seen this too - when I purposely misspell the table name the error is just an empty object `{}`. I have seen random logs with missing errors too that are intermittent.
